# Test version used by test_px4_launch.py
# Intentionally comments out and changes some values
mock_gps_node:
  ros__parameters:
    misc:
      autopilot: 'gisnav.autopilots.px4_micrortps.PX4microRTPS'
      mock_gps_selection: 1
      attitude_deviation_threshold: 10
      #max_pitch: 30
      #min_match_altitude: 80
      min_match_altitude: 70  # Non-default value
    map_update:
      update_delay: 1
      gimbal_projection: True
      max_pitch: 30
      max_map_radius: 1000
      update_map_area_threshold: 0.85
    #pose_estimator:
    #  params_file: 'config/loftr_params.yaml'
    #debug:
    #  export_position: 'position.json'
    #  export_projection: 'projection.json'